### PR TITLE
Fix edit labels modelbody css issue

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-02-15 02-19-11
Screenshot from 2023-02-15 15-04-10

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar , @GowthamShanmugam Please review

**Comment by GowthamShanmugam** [comment]
> Screenshot from 2023-02-15 02-19-11 Screenshot from 2023-02-15 15-04-10

i feel label box is bigger, Please check with UX team. but as per design this is small like text input box

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-02-16 14-09-03
Screenshot from 2023-02-16 14-06-15

File: packages/shared/modals/modal.scss [hunk 1]
```diff
@@ -21,7 +21,7 @@
   cursor: text;
   line-height: 20px;
   margin-bottom: 1rem;
-  min-height: 120px;
+  min-height: 36px;
```
Comment (SanjalKatiyar) [inline packages/shared/modals/modal.scss:24]: make sure of 2 things (and add screenshots for that as well):
1. In "Edit labels" page, this input box is still is of same height as before... we only need to decrease the height for DRPolicy related pages.
2.  After decreasing the height for DRPolicy pages, check whether older issue is not happening again (the very first issue that we solved, where adding too many labels was breaking the CSS).

Comment (TimothyAsirJeyasing) [reply packages/shared/modals/modal.scss:24]: Sure will do that

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-02-16 14-33-20
Screenshot from 2023-02-16 14-32-26
Screenshot from 2023-02-16 14-31-22
Screenshot from 2023-02-16 14-30-24

**Comment by SanjalKatiyar** [comment]
> Screenshot from 2023-02-16 14-33-20 Screenshot from 2023-02-16 14-32-26 Screenshot from 2023-02-16 14-31-22 Screenshot from 2023-02-16 14-30-24

that is what I am saying... as per ur attached SS... "Edit labels" for DRPolicy looks diff from other places...
"Edit labels" at all the places in ODF/MCO should have original height... only for DRPolicy manage/apply page we need to decrease the height for the input box...

**Comment by TimothyAsirJeyasing** [comment]
Sorry, I misunderstood that we need to reduce for edit label also. I will correct this.

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-02-16 23-23-26
Screenshot from 2023-02-16 23-22-11
Screenshot from 2023-02-16 22-19-22
Screenshot from 2023-02-16 22-18-14
Screenshot from 2023-02-16 22-17-53
Screenshot from 2023-02-16 22-17-00

**Comment by GowthamShanmugam** [comment]
LGTM, Please check once with latest master

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**